Firebase.cpp

```diff
@@ -15,17 +15,9 @@
 //
 #include "Firebase.h"
 
-// Detect whether stable version of HTTP library is installed instead of
-// master branch and patch in missing status and methods.
-#ifndef HTTP_CODE_TEMPORARY_REDIRECT
-#define HTTP_CODE_TEMPORARY_REDIRECT 307
-#define USE_ESP_ARDUINO_CORE_2_0_0
-#endif
+using std::unique_ptr;
```

Comment:
This is true, if it isn't we can drop these and fall back to comments saying "the client owns the object and is responsible for freeing" I would prefer to keep it explicit if we can though.